jwst/stpipe/cmdline.py

```diff
@@ -16,6 +18,9 @@
     'debug', 'logcfg', 'verbose'
     ]
 
+def get_cfg_from_crds(step, filename):
```

Comment:
So, I believe this should be superseded by `crds_client.get_config_from_reference` in #3922